packages/extension-chakra-storefront/src/constants.js

```diff
@@ -62,38 +28,11 @@ export const PRODUCT_BADGE_DETAILS = [
     }
 ]
 
-export const cssColorGroups = {
-    beige: '#d3bca9',
-    black: '#000000',
-    blue: '#4089c0',
-    brown: '#8e6950',
-    green: '#88c290',
-    grey: '#919191',
-    gray: '#919191',
-    silver: '#c0c0c0',
-    navy: '#000080',
-    orange: '#f4995c',
-    pink: '#f5a0ca',
-    purple: '#9873b9',
-    red: '#df5b5b',
-    white: '#FFFFFFF',
-    yellow: '#fbe85a',
-    gold: '#ffd700',
-    miscellaneous: 'linear-gradient(to right, orange , yellow, green, cyan, blue, violet)'
-}
-
-// Color to use for the UI surrounding the page in browsers.
-export const THEME_COLOR = '#0176D3'
-
-export const FILTER_ACCORDION_SATE = 'filters-expanded-index'
-
 export const API_ERROR_MESSAGE = defineMessage({
     id: 'global.error.something_went_wrong',
     defaultMessage: 'Something went wrong. Try again!'
 })
 
-export const HOME_HREF = '/'
-
 export const urlPartPositions = {
```

Comment:
This const is not likely needed to be changed in any case, it is safe to stay in this file